app/templates/template.html

```diff
@@ -18,17 +18,17 @@
         {% block headIcons %}
             <link rel="icon"
                   sizes="48x48"
-                  href="{{ assetPath | default('/assets', true) }}/images/favicon.ico">
+                  href="{{ assetPath | default('/assets', true) }}/image/favicon.ico">
             <link rel="icon"
                   sizes="any"
-                  href="{{ assetPath | default('/assets', true) }}/images/favicon.svg"
+                  href="{{ assetPath | default('/assets', true) }}/image/favicon.svg"
                   type="image/svg+xml">
             <link rel="mask-icon"
                   href="{{ assetPath | default('/assets', true) }}/images/govuk-icon-mask.svg"
                   color="{{ themeColor | default('#0b0c0c') }}">
             {#- Hardcoded value of $govuk-black #}
             <link rel="apple-touch-icon"
-                  href="{{ assetPath | default('/assets', true) }}/images/govuk-icon-180.png">
+                  href="{{ assetPath | default('/assets', true) }}/image/govuk-icon-180.png">
```

Comment:
These new icons arent the govuk-icon anymore so having them named as such may be misleading. Maybe name it something to do with tna.

On that note, I did notice we already had `image/the-national-archives-logo.svg` that we use in the webapp pages themselves, is that usuable for any of these links?

Reply:
I've replaced the govuk prefix for the files with just "tna", the ```image/the-national-archives-logo.svg```file looks like this, a little different to the ico
<img width="190" alt="image" src="https://github.com/nationalarchives/da-ayr-beta-webapp/assets/105435961/95e99d08-17a6-463c-976d-47c7839674cd">

Reply:
ah okay, thanks 🙏